wordpress-mobile/WordPress-iOS
# Add dark and light colors to ConfettiView animation

Fixes #NA

This PR adds colors for both light and dark mode to the ConfettiView animation (see examples below)

light appearance | dark appearance
|:-:|:-:|
<img height=500 src=https://user-images.githubusercontent.com/34376330/109063855-48f06300-76af-11eb-9b41-d89b3d4a1a1e.gif> | <img height=500 src=https://user-images.githubusercontent.com/34376330/109063891-54dc2500-76af-11eb-989b-b504f1297eb5.gif>




To test:

To test this PR, you'll either:
- need to point to a `sandbox` with your device or simulator, and have the backend diff for Milestone Notifications applied to it, 
- or in `Notification.swift`  edit the isViewMilestone property to match an existing notification type, e.g. a likes or "first post" milestone. In this case, in that property, you can replace note.type == "view_milestone" with note.type == "like_milestone_achievement" or with note.type == "post_milestone_achievement" (or both, if you want to test it on more notifications).
- you also need to enable the `milestoneNotifications` feature flag

Test steps:

- Build/run and go to notifications. Make sure you have some notifications of the types described in the pre-requisites
- Select a notification and make sure the confetti animation shows up and looks consistent with the examples above
- Perform the test in both light and dark appearance

PR submission checklist:


cc: @mbshakti
